ui/app/rif/components/chainAddresses.js

```diff
@@ -129,26 +153,28 @@ class ChainAddresses extends Component {
     this.setState({ insertedAddress: address });
   }
 
- timeoutToRedirect = (chainAddressesCopy, selectedChainAddress) => setTimeout(async () => {
-    let chainAddresses = await this.props.getChainAddresses(this.props.domainName, this.props.subdomainName);
-    // I need to compare the chainaddresses without the actions setted, cause the actions will be variant in time, but will send the chainaddresses to the component
-    const chainAddressesWithoutActions = chainAddresses.map(chainaddress => {
-      chainaddress.action = '';
-      return chainaddress;
-    });
-    console.debug('chainAddresses to compare', chainAddressesWithoutActions);
-    console.debug('This are the chainaddresses copied', chainAddressesCopy);
-    if (!arraysMatch(chainAddressesCopy, chainAddressesWithoutActions)) {
-      this.props.showThis(
-        this.props.redirectPage,
-        {
-          ...this.props.redirectParams,
-          newChainAddresses: chainAddresses,
-        });
-    } else {
-      this.timeouts.push(this.timeoutToRedirect(chainAddressesCopy, selectedChainAddress));
-    }
-  }, WAIT_FOR_NOTIFIER);
+ timeoutToRedirect(chainAddressesCopy, selectedChainAddress) {
+    setTimeout(async () => {
```

Comment:
Be careful you need to save this timeout identifier to the array if not your call this.timeouts.push(this.timeoutToRedirect(chainAddressesCopy, selectedChainAddress)); will not work properly when you clean the timeouts on the componentWillUnmount, i recommend you to return this timeout as a fix for this.